debian/ceph-base.postinst

```diff
@@ -33,13 +33,15 @@ case "$1" in
 	rm -f /etc/init/ceph.conf
 	[ -x /sbin/start ] && start ceph-all || :
 
-        # adjust file and directory permissions
-	for DIR in /var/lib/ceph/* ; do
-	    if ! dpkg-statoverride --list $DIR >/dev/null
-	    then
-		chown $SERVER_USER:$SERVER_GROUP $DIR
-	    fi
-	done
+	PERM_COMMAND="dpkg-statoverride --list '{}' > /dev/null || chown ${SERVER_USER}:${SERVER_GROUP} '{}'"
+
+	# adjust file and directory permissions
+	find /var/lib/ceph -mindepth 1 -maxdepth 1 -print0 \
+	    | xargs -0 -I '{}' sh -c "${PERM_COMMAND}"
```

Comment:
Why restrict find to `maxdepth 1` here and not let it recurse to cover all the files?
Or maybe do a `chown -r` to ensure all dirs and files below are owned by `cheph:ceph`?
This avoids the need to deal with `/var/lib/ceph/crash` in yet another `find + xargs`.

Reply:
I wasn't 100% sure what's appropriate here - I simply wanted to preserve the original behavior, as I wasn't certain whether something deep inside `/var/lib/ceph` would perhaps require different ownership (at some point).